fsw/cfe-core/src/sb/cfe_sb_api.c

```diff
@@ -704,6 +863,9 @@ int32  CFE_SB_SubscribeFull(CFE_SB_MsgId_t   MsgId,
     uint8  PipeIdx;
     CFE_SB_DestinationD_t *DestBlkPtr = NULL;
     char   FullName[(OS_MAX_API_NAME * 2)];
+    char   PipeName[OS_MAX_API_NAME] = {'\0'};
+
+    CFE_SB_GetPipeName(PipeName, OS_MAX_API_NAME, PipeId);
```

Comment:
I prefer to see `sizeof(PipeName)` rather than `OS_MAX_API_NAME`, as this allows the string definition to change without breaking the code.

Reply:
resolved